examples/src/utils/renderer-debug-widget.ts

```diff
@@ -373,18 +357,19 @@ function updateWantedSectorOverride(
     const acceptedDetailed = filterSectorNodes(detailedFilter, root);
     const wanted: reveal.internal.WantedSector[] = [];
     for (const node of acceptedSimple) {
+      /* TODO: j-bjorne 29-04-2020: How to add these again?
       wanted.push({
         sectorId: node.id,
         levelOfDetail: reveal.internal.LevelOfDetail.Simple,
         metadata: node
-      });
+      });*/
     }
     for (const node of acceptedDetailed) {
-      wanted.push({
-        sectorId: node.id,
+      /*wanted.push({
+        id: node.id,
         levelOfDetail: reveal.internal.LevelOfDetail.Detailed,
         metadata: node
-      });
+      });*/
```

Comment:
How would one specify a custom SectorCuller after the refac?

Reply:
RevealManager has a SectorCuller in it's constructor options, this is not exposed in the SimpleRevealManager version, but if you want it to be part of the Simple API it can be added there too.